api/client/lib/client.go

```diff
@@ -32,11 +33,35 @@ type Client struct {
 	customHTTPHeaders map[string]string
 }
 
+// NewEnvClient initializes a new API client based on environment variables.
+// Use DOCKER_HOST to set the url to the docker server.
+// Use DOCKER_API_VERSION to set the version of the API to reach, leave empty for latest.
+// Use DOCKER_CERT_PATH to load the tls certificates from.
+func NewEnvClient() (*Client, error) {
```

Comment:
that's correct, it won't be used yet, but other projects will use it for easy bootstrap. Good call about the verification, I'm going to change it.